tutor/images.py

```diff
@@ -4,8 +4,7 @@
 def build(path: str, tag: str, *args: str) -> None:
     fmt.echo_info(f"Building image {tag}")
     build_command = ["build", f"--tag={tag}", *args, path]
-    if utils.is_buildkit_enabled():
-        build_command.insert(0, "buildx")
+    build_command.insert(0, "buildx")
```

Comment:
Adding a comment about the `buildx` specifier.
```suggestion
    # Docker v19-v22 makes the new BuildKit builder available, but only if `buildx` is specified.
    # Starting in Docker v23, BuildKit is the default, and `buildx` can be omitted.
    build_command.insert(0, "buildx")
```